scripts/mainnet-fork/name.ts

```diff
@@ -0,0 +1,29 @@
+import { ChainId, registry } from '@angleprotocol/sdk/dist';
```

Comment:
What's this file?

Reply:
sth random I had developed to check the names on the EURA contract before the upgrade. Had never pushed it